src/graph/query_graph.c

```diff
@@ -740,7 +748,7 @@ GrB_Matrix QueryGraph_MatrixRepresentation
 	uint node_count = QueryGraph_NodeCount(qg_clone);
 	for(uint i = 0; i < node_count; i++) {
 		QGNode *n = qg_clone->nodes[i];
-		if(QGNode_LabelCount(n) == 0) QGNode_AddLabel(n, "", i);
+		if(QGNode_LabelCount(n) == 0) QGNode_AddLabel(n, "", i, false);
```

Comment:
please add comment explain how to get to this line

Reply:
This is reached every time we want the matrix representation of a graph with nodes with no labels (for every such node) which is reached often, since it is reached in AlgebraicExpression_FromQueryGraph whenever we have at least one edge in our query.
As this is quite generic, should I still add it?